lib/cisco_node_utils/cmd_ref/hsrp_global.yaml

```diff
@@ -0,0 +1,22 @@
+# hsrp_global.yaml
+---
+_exclude: [ios_xr]
+
+_template:
+  get_command: "show running-config hsrp all | no-more"
+
+bfd_all_intf:
+  _exclude: [N3k]
+  kind: boolean
+  get_value: '/^hsrp bfd all-interfaces$/'
+  set_value: '<state> hsrp bfd all-interfaces'
+  default_value: false
+
+extended_hold:
```

Comment:
Assuming this nvgens the same way on all nxos platforms that support this property we should really handle this property similar to `dampen_igp_metric` under bgp_af.rb.

**yaml:**
```yaml
extend_hold:
  kind: int
  get_value: '/^(no )?hsrp timers extend-hold(?:\s+(\d+))?$/'
  set_value: '<state> hsrp timers extended-hold <time>'
  default_value: 10
```

**getter:**
```ruby
def extend_hold
  match = config_get('hsrp_global', 'extended_hold', @get_args)
  if match.is_a?(Array)
    return nil if match[0][/no/]
    return match[1].to_i if match[1]
  end
  default_extend_hold
end
```
Alternatively, I would like to see if we can use the yaml `auto_default` property and write the yaml/getter as follows.

**yaml:**
```yaml
extend_hold:
  auto_default: false
  get_value: '/^hsrp timers extend-hold(?:\s+(\d+))?$/'
  set_value: '<state> hsrp timers extended-hold <time>'
  default_value: 10
```

**getter:**
```ruby
def extend_hold
  config_get('hsrp_global', 'extended_hold', @get_args)
end
```

My hope is that config_get will return the following:

1. `nil` if `no hsrp timers extended-hold` is configured.
2. `10` if `hsrp timers extended-hold` is configured.
3. `11` if `hsrp timers extended-hold 11` is configured.